### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -44,8 +44,8 @@ ram.runtime = "50M"
     [resources.sources]
 
         [resources.sources.main]
-        url = "https://github.com/penpot/penpot/archive/refs/tags/2.1.1.tar.gz"
-        sha256 = "0912464158ab8cdbef384686886de95b3318d81105522c3eeb9d703fea87fbf0"
+        url = "https://github.com/penpot/penpot/archive/refs/tags/2.1.2.tar.gz"
+        sha256 = "a61b1e57256c8070906f6cda1aee27b9b484f9e83bfdfacfecca512c0850dd02"
```
Comment (orhtej2) [inline manifest.toml:48]: Penpot has a tendency of overwriting tags, I want to wait till they officially announce the release before merging.

**Comment by orhtej2** [comment]
Note to self: Version needs to be updated in `_common.sh` as well.

**Comment by orhtej2** [comment]
!testme

**Pull request merged**